src/common/templates/_storage.tpl

```diff
@@ -19,3 +19,36 @@
   {{- end -}}
 {{- end -}}
 {{- end -}}
+{{/*
+Define a list of volumes for a pod.
+
+Usage:
+{{ include "harnesscommon.volumes" (dict "volumes" (list (dict "name" "harness-opt") (dict "name" "tmp")) ) }}
+
+Params:
+  - volumes: List - Required. List of volume definitions, each as a dict:
+      - name: String - Required.
+*/}}
+{{- define "harnesscommon.volumes" -}}
```

Comment:
These functions do not make much sense since they are pretty straightforward operations, and we only do volumes definition once in the deployment.